app/views/certificate/vatCertificate.scala.html

```diff
@@ -48,24 +48,36 @@ <h1 class="heading-xlarge">@if(user.isAgent){@messages("vatCertificate.agentTitl
             <h2 class="heading-medium card-heading">@messages("vatCertificate.aboutYourRegistration.title")</h2>
         </div>
         <div class="column-full">
-            <table>
-                <thead>
-                </thead>
-                <tbody>
-                    <tr>
-                        <td>@messages("vatCertificate.aboutYourRegistration.vrn")</td>
-                        <td>@model.vrn</td>
-                    </tr>
-                    <tr>
-                        <td>@messages("vatCertificate.aboutYourRegistration.registrationDate")</td>
-                        <td>@{model.registrationDate.map(displayDate(_)).getOrElse(messages("common.notProvided"))}</td>
-                    </tr>
-                    <tr>
-                        <td>@messages("vatCertificate.aboutYourRegistration.certificateDate")</td>
-                        <td>@displayDate(model.certificateDate)</td>
-                    </tr>
-                </tbody>
-            </table>
+            <dl class="govuk-check-your-answers cya-questions-long">
```

Comment:
You'll need to update `VatCertificateViewSpec` as most of its tests are now failing because of the CSS selectors.